cdk-project/lib/images/codebuild-image/python/src/notebooks/parse.py

```diff
@@ -6,16 +6,18 @@
 from notebooks.git import Git
 
 SKIP_LIST = {
-    "ground_truth_labeling_jobs",
     "aws_marketplace",
-    "label_data",
     "contrib",
-    "sagemaker-fundamentals",
     "end_to_end",
-    "use-cases",
-    "step-functions-data-science-sdk",
-    "prep_data",
+    "ground_truth_labeling_jobs",
     "ingest_data",
+    "label_data",
+    "prep_data",
+    "sagemaker-fundamentals",
+    "sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_example.ipynb",
```

Comment:
You might want to pick up the latest commit. The last commit https://github.com/aws/sagemaker-example-notebooks-testing/pull/12 added the directory sagemaker_edge_manager to the skip list.